app/models/event-invoice.js

```diff
@@ -26,6 +26,8 @@ export default ModelBase.extend({
   expYear      	: attr('number'),
   amount        : attr('number'),
   completedAt   : attr('moment'),
+  invoicePdfUrl : attr('string'),
+  eventId       : attr('number'),
```

Comment:
Do check if it can be achieved using `event.get('originalId')` or something similar

Reply:
this is the actual event invoice model, either the server is returning the eventId or not, if it is, then it's definition here is not optional.